.changeset/tidy-eagles-try.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
can we remove the changeset?

Reply:
Would like to summon @smrz2001 here. 

I have a suspicion, that it is here to stay with us. Maybe the right process is to fill it right before the merge.

I think you can mark the file as viewed to hide it.

<img width="248" alt="Screenshot 2022-10-20 at 21 50 51" src="https://user-images.githubusercontent.com/193527/197033170-5f98efde-81d7-4141-b8f8-cde5a35f2869.png">

Reply:
Hmm, I feel like we can probably avoid adding changesets for the time being.

We currently have two orthogonal decisions to make - how to make releases not painful + move to a more efficient package manager, the former being the more urgent, blocking issue.

If/when we decide to move to `pnpm` for the latter, having changesets will be necessary, but it seems like we can fix the release process with `lerna` without needing changesets yet.

Reply:
Having said that, do we want to move to `pnpm` next? It offers many advantages over `lerna`, but also comes with the (non-zero but also not large) extra effort of working with changesets.

I have a `pnpm` prototype PR ready, and could probably make that happen soon, but would defer to your judgement on this, @ukstv @stbrody.